std/xml.d

```diff
@@ -2714,6 +2715,16 @@ EOS";
 
 unittest
 {
+    string test_xml = `<?xml version="1.0" encoding='UTF-8'?><stream:stream
+                        xmlns:stream="http://etherx.'jabber'.org/streams"
+                        xmlns="jabber:'client'" from='jid.pl' id="587a5767"
+                        xml:lang="en" version="1.0"></stream:stream>`;
+
+    DocumentParser parser = new DocumentParser(test_xml);
```

Comment:
Should this also test that the parsed structure matches the expected structure? I.e., that the value of `encoding` is correctly parsed as `UTF-8`?